release_notes/addtl-release-notes.adoc

```diff
@@ -0,0 +1,90 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="addtl-release-notes"]
+= Additional release notes
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/servicebinding-document-attributes.adoc[]
+:context: addtl-release-notes
+
+Release notes for additional related components and products not included in the core xref:../release_notes/ocp-release-notes.adoc#ocp-release-notes[{product-title} {product-version} release notes] are available in the following documentation.
+
+[IMPORTANT]
+====
+The following release notes are for downstream Red Hat products only; upstream or community release notes for related products are not included.
+====
+
+[cols="4,7a",options="header"]
+|===
+|Component or product |Release note documentation
```

Comment:
Can you please add an entry for:

* https://docs.openshift.com/container-platform/4.17/security/cert_manager_operator/cert-manager-operator-release-notes.html

I also did a quick search for "release notes" on d.o.c., should we consider adding these as well?

* https://docs.openshift.com/container-platform/4.17/security/nbde_tang_server_operator/nbde-tang-server-operator-release-notes.html
* https://docs.openshift.com/container-platform/4.17/observability/power_monitoring/power-monitoring-release-notes.html
* https://docs.openshift.com/container-platform/4.17/observability/cluster_observability_operator/cluster-observability-operator-release-notes.html
* https://docs.openshift.com/container-platform/4.17/backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.html
* https://docs.openshift.com/container-platform/4.17/nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn.html

Reply:
Added cert-manager. Will look into the others.

Reply:
Added all.